pre_commit_hooks/util.py

```diff
@@ -13,7 +15,11 @@ def added_files() -> Set[str]:
     return set(cmd_output(*cmd).splitlines())
 
 
-def cmd_output(*cmd: str, retcode: Optional[int] = 0, **kwargs: Any) -> str:
+def cmd_output(
+    *cmd: Union[str, bytes],
```

Comment:
mixing here used to have problems on windows, I'm not sure if it's still a problem in python3 though

Reply:
Reverted. It was needed in the previous version where everything was `bytes`. I don't have Windows nearby to check at the moment, however, tests weren't failing.
They use [os.fsdecode](https://docs.python.org/3/library/os.html#os.fsdecode) ([subprocess.py#L561](https://github.com/python/cpython/blob/1642c0ef750f96664a98cadb09301d492098d2fb/Lib/subprocess.py#L561)) which, quoting the doc, _decode the path-like filename from the filesystem encoding with 'surrogateescape' error handler, or 'strict' on Windows; return `str` unchanged_ (good bad example of mixing `str` and `bytes`).